src/librustdoc/html/static/js/main.js

```diff
@@ -951,13 +951,14 @@ function hideThemeButtonState() {
             "Accepted kinds are: <code>fn</code>, <code>mod</code>, <code>struct</code>, \
              <code>enum</code>, <code>trait</code>, <code>type</code>, <code>macro</code>, \
              and <code>const</code>.",
-            "Search functions by type signature (e.g., <code>vec -&gt; usize</code> or \
+            "Search functions by type signature (e.g., <code>-&gt; usize</code> or \
              <code>* -&gt; vec</code>)",
-            "Search multiple things at once by splitting your query with comma (e.g., \
-             <code>str,u8</code> or <code>String,struct:Vec,test</code>)",
+            "You can look for functions based on its arguments (e.g., \
```

Comment:
Why does this need special syntax? There's already a "function parameters" tab.

Reply:
It's not a special syntax or anything. Maybe I badly wrote it but it's simply that you can look at function arguments specifically if you want. Extra explanations:

```
item
```

This will look for item pretty much everywhere: in the items' name, in their arguments if they have any and in the returned type (if there is any).

```
item(arg)
```

It'll now look for `item` only in item's name and will look for `arg` only in arguments.

And so this:

```
(arg)
```

Will look for `arg` only in arguments. This is what I meant here.